shared/src/main/scala/drt/shared/api/Arrival.scala

```diff
@@ -44,6 +44,24 @@ case class Arrival(Operator: Option[Operator],
     case Some(s) => s.suffix
   }
 
+  def displayStatus: ArrivalStatus = {
+
+    val fifteenMinutes = 15 * 60 * 1000
+
+    this.Estimated match {
```

Comment:
I think this code would be more expressive if you did the match on all of the fields that are considered in the logic below, it's a bit arbitrary to only match on the `Estimated` field when you are only using that in 3 of the cases.

I think you also have a bug where if a flight has an `Estimated` time you'll never get a `cancelled` or `diverted` status - you probably want to move the conditions that check the feed status to the top of the checks. (and maybe add some tests for those two cases)

we have a few examples of similar things in the codebase

```
(Estimated, Actual, ActualChox) match {
    case _ if isDiverted(Status) => "Diverted"
    case _ if isCancelled(Status) => "Cancelled"
    case (_, _, Some(actualChox) => "Landed"
    ....
}
```